packages/components/src/components/Trigger/Trigger.scss

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.bx--btn.link-btn.bx--btn--ghost {
```

Comment:
do we want to apply this globally or only in specific contexts?

Reply:
I reckon globally, how? I know we've got a vars file but it's not going to be a dollared variable is it?

Reply:
styles in App.scss apply to all pages, so we could move it there